types/backblaze-b2/backblaze-b2-tests.ts

```diff
@@ -0,0 +1,344 @@
+import BackBlazeB2 from 'backblaze-b2';
```

Comment:
```suggestion
import BackBlazeB2 =  require('backblaze-b2');
```
this is still (until 3.9) version of using CommonJS modules